src/Interpreters/executeQuery.cpp

```diff
@@ -647,7 +647,7 @@ static std::tuple<ASTPtr, BlockIO> executeQueryImpl(
 
         /// Put query to process list. But don't put SHOW PROCESSLIST query itself.
         ProcessList::EntryPtr process_list_entry;
-        if (!internal && !ast->as<ASTShowProcesslistQuery>())
+        if (!internal && !ast->as<ASTShowProcesslistQuery>() && !ast->as<ASTInsertQuery>())
```

Comment:
any special reason?

Reply:
Concurrent ingest consumes the quota which doesn't make much sense for our use cases